app/renderers/hyrax/renderers/search_and_external_link_attribute_renderer.rb

```diff
@@ -0,0 +1,40 @@
+module Hyrax
+  module Renderers
+    class SearchAndExternalLinkAttributeRenderer < AttributeRenderer
+      include ApplicationHelper
+      private
+
+      def li_value(value)
+        links_to_search_field_and_external_uri(search_field, value)
+      end
+
+      def search_field
+        options.fetch(:search_field, field)
+      end
+
+      def uri_presenter
+        options.fetch(:uri_presenter)
+      end
+
+      def links_to_search_field_and_external_uri(field, query)
+        links = []
+        uris = []
+        label_uris.each do |item|
+          uri = item.split("$").second
+          uris << uri if item.include?(query) && uri.present?
+        end
+        search_path = Rails.application.class.routes.url_helpers.search_catalog_path(
+            search_field: search_field, q: ERB::Util.h(query)
+        )
+        links << link_to(query, search_path)
+        links << link_to('<span class="glyphicon glyphicon-new-window"></span>'.html_safe, uris.first, 'aria-label' => "Open link in new window", class: 'btn btn-defaul') if uris.first.present?
+        links.join('')
+      end
+
+      def label_uris
+        ScholarsArchive::TriplePoweredService.new.fetch(uri_presenter)
```

Comment:
The label/uri are index in SOLR, so it seems like we would want to use/parse those instead of making extra calls to the TriplePoweredService to fetch the label again.